lib/Sema/CSSolver.cpp

```diff
@@ -186,6 +186,8 @@ Solution ConstraintSystem::finalize() {
 
   solution.solutionApplicationTargets = solutionApplicationTargets;
   solution.caseLabelItems = caseLabelItems;
+  solution.isolatedParams.clear();
```

Comment:
Nit: I don't think we need a to clear here because it's a brand new solution allocated by `finalize`.

Reply:
Sure, I'll drop it in a follow-up commit